service/telemetry/package_test.go

```diff
@@ -10,5 +10,8 @@ import (
 )
 
 func TestMain(m *testing.M) {
-	goleak.VerifyTestMain(m)
+	// goleak.VerifyTestMain(m)
+	// Ignore lumberjack millRun goroutine in all tests
+	goleak.VerifyTestMain(m,
```

Comment:
I think we need to add why
https://github.com/natefinch/lumberjack/issues?q=is%3Aissue%20state%3Aopen%20leak